### PR TITLE
Agent: README updates for build on ppc64le

### PR DESCRIPTION
**Pull request by jing-wang4**
README updates for agent build on ppc64le

Fix: https://github.com/kata-containers/kata-containers/issues/1069

Signed-off-by: Jing Wang <jing.wang4@ibm.com>

### CODE REVIEW

**Comment by jing-wang4** [comment]
@bpradipt @fidencio @clnperez @Amulyam24 

Please review. thanks!

File: src/agent/README.md [hunk 1]
```diff
@@ -40,9 +40,11 @@ After that, we drafted the initial code here, and any contributions are welcome.
 
 ### Build from Source
 The rust-agent needs to be built statically and linked with `musl`
+
```
Comment (clnperez) [inline src/agent/README.md:43]: nit: /s/uses/use

**Review by c3d — APPROVED** [review]
With the grammar nit (use instead of uses), lgtm

**Review by jodh-intel — APPROVED** [review]

**Comment by devimc** [comment]
thanks @jing-wang4 - CI is not happy

```
WARNING: Word 'glibc': did you mean one of the following?: glib, glib c
```

**Comment by jing-wang4** [comment]
Thanks @devimc @jodh-intel @c3d  @clnperez !

Updated accordingly, please review again.

**Comment by bpradipt** [comment]
Thanks @jing-wang4 . Hopefully the CI will be happy now. Let's see

**Review by bpradipt — APPROVED** [review]

**Pull request merged**